clickhouse/columns/string.h

```diff
@@ -19,14 +19,10 @@ class ColumnFixedString : public Column {
 
     explicit ColumnFixedString(size_t n);
 
-    template <typename Values>
-    ColumnFixedString(size_t n, const Values & values)
```

Comment:
Removing this constructor breaks client code:
```
        const auto values = std::array<const char*, 2>{
                "first row",
                "second row"};
        ColumnFixedString(16, values);
```

Reply:
just support std::vector<std::string> or std::vector<std::string_view> may be enough

Reply:
> Removing this constructor breaks client code:
> 
> ```
>         const auto values = std::array<const char*, 2>{
>                 "first row",
>                 "second row"};
>         ColumnFixedString(16, values);
> ```

Hi, this maybe the minimum change. Or, windows dll will be impossible
```c++
template<typename Values>
    static ColumnFixedString MakeColumnFixedString(size_t n, const Values& values) {
        ColumnFixedString cfs(n);
        for (const auto& v : values) {
            cfs.Append(v);
        }
        return cfs;
    }
···

It will become this:
```c++
const auto values = std::array<const char*, 2>{
                "first row",
                "second row"};
        ColumnFixedString::MakeColumnFixedString(16, values);
```
@Enmk

Reply:
another attempt :
Add __declspec(dllexport) for each related class.  It is a terrible change.